### PR TITLE
CC-116 Optimizing DataImporter speed

### PR DESCRIPTION
**Pull request by devromans**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/CC-116

###### Optional

- Core PR: https://github.com/spryker/spryker/pull/3660

###### Change log

{Relevant changes for project level go here. Those will be copied to the release log for suite if necessary.}

### CODE REVIEW

File: src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php [hunk 1]
```diff
@@ -136,6 +136,9 @@ public function collectQueueMessages(AMQPMessage $message)
     public function acknowledge(QueueReceiveMessageTransfer $queueReceiveMessageTransfer)
     {
         $this->channel->basic_ack($queueReceiveMessageTransfer->getDeliveryTag());
+        $this->publishOnRoutingKey($queueReceiveMessageTransfer);
+
+        return true;
```
Comment (dereuromark) [inline src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php]: Those should be void AFAIK?

File: src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php [hunk 2]
```diff
@@ -146,6 +149,8 @@ public function acknowledge(QueueReceiveMessageTransfer $queueReceiveMessageTran
     public function reject(QueueReceiveMessageTransfer $queueReceiveMessageTransfer)
     {
         $this->channel->basic_reject($queueReceiveMessageTransfer->getDeliveryTag(), $queueReceiveMessageTransfer->getRequeue());
+
+        return true;
```
Comment (dereuromark) [inline src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php]: Same

File: tests/SprykerTest/Zed/RabbitMq/Business/Model/Exchange/Filter/ExchangeFilterByNameTest.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * MIT License
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
```
Comment (tamasnyulas) [inline tests/SprykerTest/Zed/RabbitMq/Business/Model/Exchange/Filter/ExchangeFilterByNameTest.php]: wrong licence, revert

File: tests/SprykerTest/Zed/RabbitMq/_support/RabbitMqBusinessTester.php [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * MIT License
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
```
Comment (tamasnyulas) [inline tests/SprykerTest/Zed/RabbitMq/_support/RabbitMqBusinessTester.php]: wrong licence, revert

File: tests/SprykerTest/Zed/RabbitMq/Business/Model/Exchange/Filter/ExchangeFilterByNameTest.php (hunk 3 above)
Comment (dereuromark) [reply tests/SprykerTest/Zed/RabbitMq/Business/Model/Exchange/Filter/ExchangeFilterByNameTest.php]: See https://travis-ci.org/spryker/rabbit-mq/builds/415906027

**Pull request merged**